include/alpaka/mem/buf/BufCpu.hpp

```diff
@@ -244,10 +244,10 @@ namespace alpaka
                 using Allocator = AllocCpuAligned<std::integral_constant<std::size_t, alignment>>;
                 static_assert(std::is_empty_v<Allocator>, "AllocCpuAligned is expected to be stateless");
                 auto* memPtr = alpaka::malloc<TElem>(Allocator{}, static_cast<std::size_t>(getExtentProduct(extent)));
-                auto deleter = [queue = std::move(queue)](TElem* ptr) mutable
+                auto deleter = [l_queue = std::move(queue)](TElem* ptr) mutable
```

Comment:
`icpx` complains about lambda `queue` shadowing function `queue` here, hence the change.